conformance/runner/run_cv_bundle.py

```
@@ -1403,6 +1403,36 @@ def validate_vector(
             )
         if "expect_block_hash" in v and go_resp.get("block_hash") != v["expect_block_hash"]:
             problems.append(f"{gate}/{vid}: expect_block_hash mismatch")
+    elif op == "block_basic_check_with_fees":
+        if go_resp.get("block_hash") != rust_resp.get("block_hash"):
+            problems.append(
+                f"{gate}/{vid}: block_hash mismatch go={go_resp.get('block_hash')} rust={rust_resp.get('block_hash')}"
+            )
+        if "expect_block_hash" in v and go_resp.get("block_hash") != v["expect_block_hash"]:
+            problems.append(f"{gate}/{vid}: expect_block_hash mismatch")
+    elif op == "connect_block_basic":
+        def as_int(x: Any) -> int:
```

Ruff (0.15.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `x`

(ANN401)